### DISCUSSION TITLE
MoBi: Modifying a distribution model

### DISCUSSION THREAD

**Discussion by dzianismr** [Open-Systems-Pharmacology/Forum#1473] [Questions & Problems] [answered]
How do I modify a distrbution model in MoBi?

In spatial structure there formulas of a type:
K_water_cell = Neighborhoods|Muscle_int_Muscle_cell|MOLECULE|Partition coefficient (intracellular/water)
which eventually points to Partition coefficient (intracellular/plasma)
![image](https://user-images.githubusercontent.com/47644764/224092670-2f40d239-8769-432f-8629-cf95c4bfb5b5.png)
which are not "normal" mobi formula.
In the simulation applied formula can be seen, but cannot be modified. I guess my question boils down to how to edit "PARTITION COEFFICIENT FORMULA".

**Comment by StephanSchaller** [comment, marked as answer]
Hi @dzianismr,

the partitioning method can be changed on the properties tab of the molecule in the molecule BB:
![image](https://user-images.githubusercontent.com/25477643/224107433-32187b96-c0c5-447b-9fa6-76334d132616.png)


Best,
Stephan

**Reply by dzianismr** [reply]
Hi @StephanSchaller,

Thanks for pointing this out. Where do I go if I would like to modify one of the partition methods ?

Best Regards, Denis

**Reply by StephanSchaller** [reply]
You can find them here, usually: C:\ProgramData\Open Systems Pharmacology\MoBi\X.Y\AllCalculationMethods.pkml

We have some experience working with this when we expanded this file with a new method for our Birds Project (https://github.com/Open-Systems-Pharmacology/Birds-PBK-Model). 
So we are happy to provide you support if required.

**Reply by dzianismr** [reply]
Thanks Stephan
